nuts/Issues/agb/description.xml

```diff
@@ -29,21 +29,20 @@
 		<nl>Zorg</nl>
 	</Category>
 	<FAQIntro>
-		<en></en>
-		<nl>Zet je AGB-code in je IRMA-app.</nl>
+		<en>Add your AGB code to your IRMA app.</en>
+		<nl>Laad je AGB-code in je IRMA-app.</nl>
     </FAQIntro>
 	<FAQPurpose>
-		<en></en>
+		<en>With your AGB code in your IRMA app, you can prove what your healthcare provider code in the AGB registry is.</en>
 		<nl>Met jouw AGB-code in de IRMA-app kun je bewijzen wat je code is als zorgverlener in het AGB-register.</nl>
 	</FAQPurpose>
 	<FAQContent>
-		<en></en>
-		<nl>Je haalt de volgende gegevens op:\n• AGB-code</nl>
+		<en>You will add the following data.\n• AGB code</en>
+		<nl>Je laadt de volgende gegevens.\n• AGB-code</nl>
 	</FAQContent>
 	<FAQHowto>
-		<en></en>
-		<nl>Doorloop de volgende stappen:\n1. Geef je initialen, achternaam en geboortedatum vrij aan NUTS.\n2. Vul je AGB-code in.\n3. Bevestig het overzetten van je AGB-code naar je IRMA-app.\n\nJe ontvangt een kaartje met je AGB-code.
-</nl>
+		<en>In case you are a healthcare professional registered in the Netherlands, you can load your AGB code into your IRMA app. But first, you need to load your personal data, stemming from the Dutch population registry. Assuming you already have these personal data in your IRMA app, complete the following steps.\n1. Reveal your initials, surname and date of birth to NUTS via IRMA.\n2. Enter your own AGB code manually.\n3. Confirm that you wish to add your AGB code to your IRMA app. After verifying your data in the AGB registry, NUTS copies your AGB code into your IRMA app.\n\nYou now receive a card with your AGB code.</en>
```

Comment:
Nuts capitalizes, but does not use full-caps for their own name. I would suggest to avoid "stemming" by just leaving out the word entirely.

Reply:
@bajacobs I would prefer "Nuts" rather than leaving it out. What do you think?

Reply:
On their website nuts.nl, they describe themselves as "stichting Nuts". I think we should use that, just like we use "stichting Privacy by Design". In English this would be "Nuts foundation"